src/gwt/src/org/rstudio/core/client/AnsiCode.java

```diff
@@ -141,28 +144,28 @@ public static class BackColorNum
 
    public static class ForeColor
    {
-      public static final String BLACK        = CSI + RESET + ";30" + SGR;
```

Comment:
Let me know if I should revert these changes -- technically, `SGR` refers to the sequence `CSI <n> m`, or `<ESC> [ <n> m`, so just binding it to `"m"` feels awkward.